src/models/user.js

```diff
@@ -15,6 +15,10 @@ userSchema.methods.saveUser = function saveUser(callback) {
   return this.save(callback);
 };
 
-const Model = mongoose.model(modelName, userSchema);
+userSchema.methods.loginUser = function loginUser(callback) {
+  return this.findOne(callback);
```

Comment:
`findOne` needs the `conditions` argument to be passed so it knows which group to find.